spec/spec/spec_helper.rb

```diff
@@ -0,0 +1,89 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
This file seems odd.  Why is it spec/spec? Can't we just use the spec/spec_helper.rb that's already there?